app/controllers/api/publishers_controller.rb

```diff
@@ -65,6 +75,16 @@ def require_valid_legal_form
     end
   end
 
+  def require_publisher
+    @publisher = Publisher.find_by(brave_publisher_id: params[:brave_publisher_id])
```

Comment:
There can be multiple publishers per `brave_publisher_id` which have `verified: false`. For example, many curious people have tried to register `google.com`, creating many unverified Publisher records. The only constraint is there can only be 1 verified publisher.

Depending on what @mrose17 feels I think this should destroy all Publisher records matching the brave_publisher_id.